blog/2025-07-30-dtg-july-update/index.md

```diff
@@ -0,0 +1,91 @@
+* Optimism's [Retro Funding: Dev Tooling](https://atlas.optimism.io/missions/retro-funding-dev-tooling) (until end of 2025) is supporting member projects: [Solidity](https://atlas.optimism.io/project/0xcc8d03e014e121d10602eeff729b755d5dc6a317df0d6302c8a9d3b5424aaba8), [Vyper](https://atlas.optimism.io/project/0x9ca1f7b0e0d10d3bd2619e51a54f2e4175e029c87a2944cf1ebc89164ba77ea0), [alloy](https://atlas.optimism.io/project/0x56ce7cbc27852a8d8ef5869dc9033a215c8893f799468f61527dacb9f92be790), [ethers.js](https://atlas.optimism.io/project/0xa3d07f453f70d844196d89d79848aa2e70a0bd8b38bf0f493cba1547bb3bca5e), [Nethereum](https://atlas.optimism.io/project/0x4a5e771af86cf1938056b43cddbf0018dca1376d578f631f7449fe10ac4958ed), [viem](https://atlas.optimism.io/project/0x6bd057da522918a4675396313ae33a2f2788a1ceeb3bd7ae228015e3eb317a7d), [web3.py](https://atlas.optimism.io/project/0xebe03c3d6d33cad60124b9b05ef6e2ff056293a1de3c5fa51dfbb90c86c14bf7), [Ape](https://atlas.optimism.io/project/0xa0b16714baef75d97ec07fd48eaf42e79df92fe2a3c2d725d2388ede587ea54c), [Foundry](https://atlas.optimism.io/project/0x4562c0630907577f433cad78c7e2cc03349d918b6c14ef982f11a2678f5999ad), [Scaffold-ETH](https://atlas.optimism.io/project/0x154a42e5ca88d7c2732fda74d6eb611057fc88dbe6f0ff3aae7b89c2cd1666ab) and [Sourcify](https://atlas.optimism.io/project/0x51cda5996ef1a2ccd8fcf4ee5792337695599454c83eb1218c3ad4388dcb5bf5)
+* Golem Foundation's [Octant Epoch 8](https://octant.app/home) supported member projects: [Solidity](https://octant.app/project/8/0xe2F7cF9C2b12c0BfcdAB571F9E50418fC08F4AD1), [Vyper](https://octant.app/project/8/0x0b450A3688a55d4221329D31e2F103bCe9adAc40), [ethers.js](https://octant.app/project/8/0x8ba1f109551bD432803012645Ac136ddd64DBA72) and Scaffold-ETH (via [BuidlGuidl](https://octant.app/project/8/0x6A4E5ed62d3827128Dbd70c5bDe25C0e6c6aA537))
+
+### Smart Contract Language
+#### [Solidity](https://soliditylang.org/) 
+* Celebrating [10 years of Solidity](https://x.com/solidity_lang/status/1942905422467551248).
+* Learn about current priorities and upcoming changes in the weekly [Language Design calls](https://meet.solidity.org) (Wednesdays 13:00 UTC).
+
+#### [Vyper](https://vyperlang.org/)
+
+### Client Library
+#### [alloy](https://alloy.rs/)
+* alloy [1.0.18](https://github.com/alloy-rs/alloy/releases/tag/v1.0.18) - [1.0.23](https://github.com/alloy-rs/alloy/releases/tag/v1.0.23)
+
+#### [ethers.js](https://ethers.org/)
+#### [Nethereum](https://nethereum.com/)
+#### [viem](https://viem.sh/)
+* viem [2.31.7](https://github.com/wevm/viem/releases/tag/viem%402.31.7) - [2.33.0](https://github.com/wevm/viem/releases/tag/viem%402.33.0): includes [initial support for Flashblocks](https://x.com/_jxom/status/1946961084125954476) (up to 200ms transaction preconfirmations)
+
+#### [web3.py](https://web3py.readthedocs.io/)
+* web3.py [v7.12.1](https://web3py.readthedocs.io/en/latest/release_notes.html#web3-py-v7-12-1-2025-07-14)
+
+#### [Web3j](https://docs.web3j.io/)
+
```

Comment:
* Two mentorships have started and are currently ongoing, [blog with details](https://www.lfdecentralizedtrust.org/blog/web3j-mentorship-2025-meet-the-mentees)